assets/src/edit-story/karma/fixture.js

```diff
@@ -337,10 +342,12 @@ class ComponentStub {
   }
 }
 
-function HookExecutor({ hooks, children }) {
+/* eslint-disable react/prop-types, react/jsx-no-useless-fragment */
+function HookExecutor({ hooks }) {
   hooks.forEach((func) => func());
-  return children;
+  return <></>;
```

Comment:
Can't you just return `null`?

Reply:
Yes, but there are warnings all over the place. So I figured an extra fragment is better than dozens of warnings in the console.